sdks/universal-router-sdk/src/utils/pathCurrency.ts

```diff
@@ -0,0 +1,36 @@
+import { Currency, CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { Pool as V4Pool } from '@uniswap/v4-sdk'
+import { TPool } from '@uniswap/router-sdk/dist/utils/TPool'
+
+export function amountWithPathCurrency(amount: CurrencyAmount<Currency>, pool: TPool): CurrencyAmount<Currency> {
```

Comment:
where is this used?

Reply:
this file was a copy paste from router-sdk bc I forgot to export stuff. can delete, but eventually Im just gonna delete this file entirely (just didn't wanna wait on the chain of approvals)